### ISSUE TITLE
Website can't be reached 

### ISSUE THREAD

**Issue by swaradgat19** [alshedivat/al-folio#1898]
Hi. I'm facing a peculiar issue with my website. I had forked the repo, added my data and setup my website on `<username>.github.io`. It was working a couple of months back, but when I checked it today, it wasn't working. I put in a commit and pushed the changes to see if it works, but it didn't. I'm getting a `This site can’t be reached` error. 

Could you tell me why this may happen? I read up about GitHub pages and if I'm exceeding any limits (there's a 1 GB limit), but I do not think I'm violating any of those rules.

**Comment by george-gca** [comment]
Just checked your site and it is working as expected. Have you found the problem?

**Comment by swaradgat19** [comment]
Yes. I figured out the problem. I am currently using university wifi and it is flagging my website and blocking it. I used mobile data to access my website and it worked
